### PR TITLE
VACMS-14701 Update vba facility fields

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

Relates to #14701

## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

As an admin
1. go to https://pr15143-x90pmpzpeuow6pi4uyluoagtj5rpzncy.ci.cms.va.gov/salt-lake-city-va-regional-benefit-office-at-george-e-wahlen-department-of-veterans-affairs-medical-center/locations/pre-discharge-site-at-fort
   - [x] Validate that you see DB data fields in the "Why can't I edit this gray box.
   - [x] Validate the order 
      - Title
      - operating status
      - page intro and photo (those are empty so not showing on node view)
      - Facility data 
      - Prepare for your visit (empty so not showing)

2. Click the edit tab
   - [x] Validate that the Non-VA location official name, Non-VA location URL, Parent office and Shared VHA location field appear in the Facility Data box.  
   - [x] validate that since you are admin you can actually edit most of the fields  (not the Name of facility and locator api id those are ultra locked)
   - [x] Validate the order 
      - Title
      - operating status
      - page intro and photo 
      - Facility data 
      - Prepare for your visit 
      - editorial workflow
3. Change the status and save the node as draft
   - [x] validate that the change of status happens.

Login as MicheleMid
5. Go to https://pr15143-x90pmpzpeuow6pi4uyluoagtj5rpzncy.ci.cms.va.gov/salt-lake-city-va-regional-benefit-office-at-george-e-wahlen-department-of-veterans-affairs-medical-center/locations/pre-discharge-site-at-fort
6. Click the edit tab  
- [x] Validate that you see all the facility api data and db data but can not edit it
- [x] validate that you can change the status and save the node successfully. 

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by swirtSJW** [comment]
The DB data is not locking down correctly.

File: config/sync/paragraphs.paragraphs_type.health_care_local_facility_servi.yml [hunk 1]
```diff
@@ -3,7 +3,8 @@ langcode: en
 status: true
 dependencies: {  }
 id: health_care_local_facility_servi
-label: 'VAMC facility service (non-healthcare service)'
```
Comment (swirtSJW) [inline config/sync/paragraphs.paragraphs_type.health_care_local_facility_servi.yml:6]: This is used across vet centers, VBA and VAMC so I altered the name to be less specific.

**Review by omahane — CHANGES_REQUESTED** [review]
It appears that the order of the fields in the node:view is not as expected from the QA:
Screenshot 2023-09-11 at 8 47 38 AM

It also appears that the order of the fields in node:edit is not as expected from QA:
Screenshot 2023-09-11 at 8 52 52 AM

**Review by omahane — APPROVED** [review]
Everything looks good to me.

**Comment by swirtSJW** [comment]
image

**Pull request merged**